src/AppBar/AppBar.js

```diff
@@ -39,12 +39,10 @@ export default class AppBar extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
-    primary: PropTypes.bool,
   };
 
   static defaultProps = {
     accent: false,
-    primary: true,
```

Comment:
Yeap, my comment is out of the scope of this PR. I would take the Button component as an example. `accent`, `primary` and `contrasted` are mutually exclusive. Having them under three property can only end badly. I would rather have a `PropTypes.oneOf(['accent', 'primary', 'contrasted', ''])` property.